readme.md

```diff
@@ -116,7 +138,7 @@ How is rebase different than a merge?
 
 Your Answer:
 ```text
-
+Rebase is keeping develpment stream line in one line, which keeps tidy. In order to do that, it erases the different branch and merges.
```

Comment:
:+1: Specifically a rebase is applying the changes of a branches commit history to the head of another branch.